docs/building-rnw.md

```diff
@@ -33,7 +33,7 @@ Then
 
 ```cmd
 cd packages\playground
-react-native run-windows
+react-native run-windows --sln windows\playground.sln
```

Comment:
The sln option isn't needed in a normal cli app is it?  Why is it needed for playground?

Reply:
it isn't for a normal app. Playground has both a playground-win32.sln and a playground.sln in the windows folder, so if you just do run-windows you used to get one picked at mostly random (whatever the glob gave back, we indexed at 0). I fixed that to detect the case of multiple SLNs and break with a message to pick one and pass it as a param to `--sln`.